TypeCobol/Compiler/CupParser/TypeCobolProgram.cup

```diff
@@ -1432,7 +1432,9 @@ rewriteStatement ::= RewriteStatement:stmt
 	;
 
 searchStatementWithBody ::=
-		searchStatement whenSearchConditionClauses
+        searchStatement SearchStatementEnd:end
```

Comment:
Indent: could you fix the indentation so as to preserve alignment with next lines ?

Reply:
Also another remark, when modifying `TypeCobolProgram.cup`, we should also report the changes into `EmptyTypeCobolProgram.cup` without the parser call `my_parser.Builder.EndSearchStatementWithBody(end);`. `EmptyTypeCobolProgram.cup` is an empty grammar file to help create another parser, it is not currently used but should be kept in-sync.

Reply:
Fixed in ebdc2b7f2d56b9226571d1def1198cd2e6b439af